tests/acceptance/README.md

```diff
@@ -0,0 +1,36 @@
+# Acceptance Tests
+
+The following BATs tests can be used to test basic functionality of the GCP Auth Engine.
+
+## Prerequisites
+
+* Clone this repository to your workstation
+* [Bats Core installed](https://bats-core.readthedocs.io/en/stable/installation.html#homebrew)
+* Docker
+* Vault CLI installed
+* GCP Project that has a service account w/ [required permissions](https://www.vaultproject.io/docs/auth/gcp#required-gcp-permissions)
+
+### GCP Testing
+
+First, set the following env variables from your GCP project 
+
+* SERVICE_ACCOUNT_ID
+* PATH_TO_CREDS env variable pointing to service account credentials JSON file
+* GOOGLE_APPLICATION_CREDENTIALS
+* GOOGLE_PROJECT
```

Comment:
Is this the ID or name of the project?

Reply:
This is the Project ID, I can add that to the docs

Reply:
Can we just change the name here to `GOOGLE_PROJECT_ID` for clarity?